hbase-server/src/test/java/org/apache/hadoop/hbase/master/TestSplitWALManager.java

```diff
@@ -86,6 +99,59 @@ public void teardown() throws Exception {
     TEST_UTIL.shutdownMiniCluster();
   }
 
+  @Test
+  public void testWALArchiveWithDifferentWalAndRootFS() throws Exception{
+    HBaseTestingUtility TEST_UTIL_2 = new HBaseTestingUtility();
```

Comment:
Since this is not class level constant, no need to keep it in capital letters, keep it `test_util_2`